packages/trip-details/src/index.tsx

```diff
@@ -232,12 +232,6 @@ export function TripDetails({
     walkDuration
   } = coreUtils.itinerary.calculatePhysicalActivity(itinerary);
 
-  const co2 = coreUtils.itinerary.calculateEmissions(
-    itinerary,
-    co2Config?.carbonIntensity,
-    co2Config?.units
-  );
-
```

Comment:
Do you think it's a good idea to still do co2 calculation if `carbonIntensity` is defined but `itinerary.co2` isn't?

Reply:
Sure! I can add this back in with a check.